volumina/widgets/layerwidget.py

```diff
@@ -207,9 +213,28 @@ def __init__( self, parent=None ):
         self.bar.fractionChanged.connect( self._onFractionChanged )
         self.toggleEye.activeChanged.connect( self._onEyeToggle )
         self.channelSelector.valueChanged.connect( self._onChannelChanged )
+        self.setUpShortcuts()
 
-    def mousePressEvent( self, ev ):
-        super(LayerItemWidget, self).mousePressEvent( ev )
+    def setUpShortcuts(self):
+        mgr = ShortcutManager()
+        ActionInfo = ShortcutManager.ActionInfo
+
+        selector = self.channelSelector
+
+        def inc():
+            selector.setValue(selector.value() + selector.singleStep())
+
+        def dec():
+            selector.setValue(selector.value() - selector.singleStep())
+
+        # Can't pass channelSelector(QSpinBox) as tooltip widget
+        # because it doesn't have # separate tooltips for arrows
+        mgr.register(NEXT_CHANNEL_SEQ, ActionInfo(
+            "Channels", "Next channel", "Next channel", inc, selector, None
```

Comment:
I think channel switching should be in the "Navigation" category.

Reply:
Updated